azurerm/resource_arm_cosmos_db_account.go

```diff
@@ -77,6 +77,26 @@ func resourceArmCosmosDBAccount() *schema.Resource {
 				),
 			},
 
+			"is_virtual_network_filter_enabled": {
+				Type:     schema.TypeBool,
+				Optional: true,
+				Default:  false,
+			},
+
+			"virtual_network_rules": {
```

Comment:
as discussed this shouldn't be nested like this.

```go
           "virtual_network_rule_ids": {
                Type:     schema.TypeSet,
				Optional: true,
                Elem:     &schema.Schema{Type: schema.TypeString},
				Set:      schema.HashString,
           }
```

Reply:
I tried that and it didn't work, I can try again...